analyzers/its/config/NetCore31WithConfigurableRules/SonarLint.xml

```diff
@@ -11,6 +11,25 @@
         </Parameter>
       </Parameters>
     </Rule>
+    <!-- Once the correct location mapping for razor generated files is implemented the following two configurations can be removed. -->
```

Comment:
If all else is good I will open an issue before merging the PR to make sure this will be removed as part of the work when the mapping is done.